shared/utils.js

```diff
@@ -3,7 +3,12 @@
 // TODO: make all of these functions configurable from the outside
 // the user should be able to manipulate those
 
-function getNamespace(service, stage) {
+function getNamespace(serverless, stage) {
+  const { namespace } = serverless.service.provider.k8s
+  const { service } = serverless.service
```

Comment:
This can probably be resolved after following condition (only when necessary)

Reply:
sure, done!